src/dapla_pseudo/v1/builder.py

```diff
@@ -138,9 +142,10 @@ def __init__(
             self._metadata: t.Dict[str, str] = {}
             self._pseudo_keyset: Optional[PseudoKeyset] = None
 
-        def map_to_stable_id(self) -> Self:
+        def map_to_stable_id(self, version_timestamp: Optional[str] = None) -> Self:
```

Comment:
We need a detailed description of what the format of this argument should be and where the user could potentially find it.